app/views/play/level/PlayLevelView.coffee

```diff
@@ -808,8 +827,12 @@ module.exports = class PlayLevelView extends RootView
     return if @destroyed
     Backbone.Mediator.publish 'level:set-time', ratio: 1
     return if @level.hasLocalChanges()  # Don't award achievements when beating level changed in level editor
-    if @goalManager.checkOverallStatus() is 'success'
-      showModalFn = -> Backbone.Mediator.publish 'level:show-victory', showModal: true
+    state = @session.get('state') ? {}
+    options = { showModal: true }
+    if @goalManager.finishLevel()
+      if state.capstoneStage and @goalManager.getRemainingGoals().length == 0
```

Comment:
Do we still set `state.complete = true` in the session for capstone levels?

Reply:
Updated - thank you for catching this!